### ISSUE TITLE
Ruby 2.7.1 - URI.escape is obsolete

### ISSUE THREAD

**Issue by myxoh** [muxinc/mux-ruby#12]
Here's an article explaining why the method was deprecated and some solutions:

https://docs.knapsackpro.com/2020/uri-escape-is-obsolete-percent-encoding-your-query-string

**Comment by dylanjha** [comment]
@myxoh @fidalgo @nikz 

I'm working on a patch here, but want to make sure I don't have a regression. Can you try this in your Gemfile:

```
gem 'mux_ruby', :git => 'git://github.com/muxinc/mux-ruby.git', :branch => 'dj/uri-escape'
```

Please only do this in development and/or in testing -- DO NOT do this in production without extreme caution. Please let me know:

1. Does this fix the deprecation warning in Ruby 2.7.1
2. Do you find any regressions? In other words, do all your API calls still work as expected?

Thanks for surfacing this issue!

**Comment by nikz** [comment]
Works fine for me - clears the deprecation warning, and no regressions that I can see immediately!

**Comment by dylanjha** [comment]
This was released in 1.5.1

https://rubygems.org/gems/mux_ruby

**Comment by nikz** [comment]
Thanks @dylanjha !
